pkg/platform/packet/packet.go

```diff
@@ -288,6 +294,46 @@ func createTerraformConfigFile(cfg *config, terraformPath string) error {
 	return nil
 }
 
+func validateNodePrivateCIDRs(cfg *config) error {
+	// If both the deprecated and the new fields are provided then error out.
+	if cfg.NodePrivateCIDR != "" && len(cfg.NodePrivateCIDRs) > 0 {
+		//nolint:stylecheck
+		return fmt.Errorf("`node_private_cidr` and `node_private_cidrs` cannot be used together." +
+			" Use `node_private_cidrs` only.")
```

Comment:
I think we can remove the dot here and remove `nolint:stylecheck` then.

Reply:
There are two sentences. Just removing one dot will not be enough, the linter will complain still.

Reply:
Oh I saw the next comment now.